dex/price-discovery/src/lib.rs

```diff
@@ -206,9 +206,10 @@ pub trait PriceDiscovery:
         );
     }
 
-    /// After the liquidity pool has been created and the LP tokens received,
-    /// users can withdraw their fair share of the LP tokens by depositing their SFTs
-    /// and a share of the extra rewards
+    /// After the unbond period has ended,
```

Comment:
this has to be changed. as not after unbound - but after initial price discovery phase

Reply:
In the next PR :)
The current description was made to match the current implementation